libs/agno/agno/workflow/utils.py

```diff
@@ -0,0 +1,19 @@
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class WorkflowAction(Enum):
```

Comment:
Should we have all 3 actions? Why not just have a single top-level bool for `run_workflow` or `continue_to_workflow`? Because it doesn't look like there is a difference between respond_directly and ask_for_more_information?